### PR TITLE
Added intended audience, purpose, goals/non-goals

### PR DESCRIPTION
**Pull request by mwherman2000**
Addresses:
- https://github.com/w3c/vc-data-model/issues/813
- https://github.com/w3c/vc-data-model/issues/792 (this time from the 1.1 branch)
- https://github.com/w3c/vc-data-model/pull/800#issuecomment-916302702

***
<a href="https://pr-preview.s3.amazonaws.com/mwherman2000/vc-data-model/pull/814.html" title="Last updated on Nov 13, 2021, 7:07 PM UTC (6ba115d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/814/ea2a3ab...mwherman2000:6ba115d.html" title="Last updated on Nov 13, 2021, 7:07 PM UTC (6ba115d)">Diff</a>

### CODE REVIEW

File: index.html
Comment (mprorock) [inline index.html:1287]:
```suggestion
        
```
What does this section have to do with "intended audience, purpose, goals" etc.  I suggest this paragraph

**Review by mprorock — CHANGES_REQUESTED** [review]
While some of this language I think might be helpful, a lot of this PR appears to restate the purposes and goals of w3c standards and audiences for those standards as detailed on the main [standards](https://www.w3.org/standards/) page itself and on related links from that page.

This PR is also missing appropriate `<a>` tags into definitions as well as has some terminology issues that might cause confusion to some audiences such as "verifiable credential technology" as opposed to something like:
`systems that utilize the Verifiable Credential<a href="#core-data-model">data model</a>`

Seems like it might be better to move this PR to a draft and get some feedback in...

File: index.html
Comment (dlongley) [inline index.html:261]: Hmm, I read the meaning of this as "the intended audience for the VC data model spec is people interested in using VC technology" -- which, to me, seems circular and -- therefore, doesn't really add anything for the reader.

I'd think the intended audience would be people that want to express physical or digital documents that include assertions about other people, places, or things in a standard digital format so that they can be more widely consumed -- and so that the documents' authorship can be verified by third parties (verifiers). Or something along those lines. However, this sort of text may already be gleaned from the spec -- so it may just be a restatement of that.

File: index.html
Comment (dlongley) [inline index.html:265]: I'm not sure that this is actually the primary audience. At least, I'd think we'd be highlighting credential authors and consumers more as opposed to "software architects" and "UX specialists". If I were someone that needed to figure out how to express credential information for some use case or to know how my application will consume it/which parts my application may need to consume (or how to build verification software), this is the spec for me. What do others think?

File: index.html
Comment (dlongley) [inline index.html:277]: I think the purpose of the spec is to tell readers how to express digital credentials in a standard format (and things along those lines), not tell people how to use VC technology. I'm also not sure that saying that the spec should give people a "deeper understanding of what a verifiable credential is" makes sense -- the spec defines what a VC is. We could say that the purpose of the VC spec is to define what a VC is ... but that's also perhaps unnecessary or self-explanatory.

File: index.html
Comment (dlongley) [inline index.html:286]: "transmission" may be over stepping here; it's a data model spec and "transmission" moves into protocol which was intentionally avoided and not in the charter when the spec was written.

File: index.html
Comment (dlongley) [inline index.html:299]: I think linking to the use cases document may be better (and I think/hope we already do so).

**Review by dlongley — COMMENTED** [review]
I think there's a lot that the WG would need to discuss here to ensure it is an accurate reflection of the spec's goals, audience, etc. and that there is consensus on those things. My initial read is that it suggests a much more ambitious spec than it actually is.

File: index.html
Comment (mwherman2000) [reply index.html:1287]: > What does this section have to do with "intended audience, purpose, goals" etc. I suggest this paragraph gets pulled, and that where possible we look at one topic per PR

Killing 2 birds with one stone @mprorock per https://github.com/w3c/vc-data-model/pull/800#issuecomment-916302702

File: index.html
Comment (mwherman2000) [reply index.html:261]: > Hmm, I read the meaning of this as "the intended audience for the VC data model spec is people interested in using VC technology" -- which, to me, seems circular and -- therefore, doesn't really add anything for the reader.

@dlongley What aspect do you think is _circular_?

> I'd think the intended audience would be people that want to express physical or digital documents that include assertions about other people, places, or things in a standard digital format so that they can be more widely consumed -- and so that the documents' authorship can be verified by third parties (verifiers). Or something along those lines. However, this sort of text may already be gleaned from the spec -- so it may just be a restatement of that.

@dlongley I agree that your comments represent a deeper level of detail. I have no problem adding something like:

> More specifically, the primary audience is software architects and developers that want to express physical or digital documents that include assertions about other people, places, or things in a standard digital format so that they can be more widely consumed.

...or something like that.  I'll wait before making this change.

File: index.html
Comment (mwherman2000) [reply index.html:265]: > highlighting credential authors and consumers more as opposed to "software architects" and "UX specialists". 

Who are these people in real life? ...that is, _credential authors_ and _consumers_.  Who are these people on your VC-based app project team? I think we need words (audience roles) that identify with real people and their roles. Further, _credential authors_ and _consumers_ sound like users of a verifiable credential-based system ...not designers and builders of VC-based systems.  The later IMO is who we're targeting. Correct?

> If I were someone that needed to figure out how to express credential information for some use case or to know how my application will consume it/which parts my application may need to consume (or how to build verification software), this is the spec for me. What do others think?

The answer is actually "no" IMO.  The proposed Guide (in addition to the Documentation consisting of the VC data model recommendation, the VC use cases note, and the other 3rd document whose formal name escapes me) is what people (architects and developers) need to read.

Aside: Perhaps _business analysts_ could or should be added to the intended audience statement?

Caution: Think of the intended audience statement as a commitment or contract with the intended readers of the Recommendation. So, for example, if we chose to add any role (e.g. business analyst), we need to make sure there is sufficient content of sufficient relevance and depth to satisfy readers assigned to the particular role.  

For v1.1 specifically, I believe we want to err on the side of a limited set of roles at this point in the review/approval process.

File: index.html
Comment (mwherman2000) [reply index.html:277]: > We could say that the purpose of the VC spec is to define what a VC is

Good feedback. The PR was intended to be a first draft - to be improved along these lines.

Aside: My original wording assumed the reader would be arriving to read the Documentation with some level of awareness of what a VC is.  I have no problem simplifying this sentence.  I'll wait and try to incorporate all of the changes at the same time.

File: index.html
Comment (mwherman2000) [reply index.html:286]: > transmission

I agree ...as a writing style, I like to include lists of things of length 3 ...they tend to be easier to read.  

Is there a third term we can use along with "creation" and "verification" in this sentence?

**Comment by mwherman2000** [comment]
> While some of this language I think might be helpful, a lot of this PR appears to restate the purposes and goals of w3c standards and audiences for those standards as detailed on the main [standards](https://www.w3.org/standards/) page itself and on related links from that page.

Thinking with the mindset of the intended audience (intended reader), virtually no one reads the top-level documents.  They google for any sort of documentation related to a topic (e.g. verifiable credentials) and start reading from there.

This is an example of why the _intended audience statement_ is so important - it causes you to think more specifically about how your targeted reader (aka persona) thinks and acts when looking for information.

File: index.html
Comment (dlongley) [reply index.html:261]: @mwherman2000,

>> Hmm, I read the meaning of this as "the intended audience for the VC data model spec is people interested in using VC technology" -- which, to me, seems circular and -- therefore, doesn't really add anything for the reader.
>>
> @dlongley What aspect do you think is circular?

To me, it is saying the intended audience of a spec that defines X are people that are interested in X. This seems largely vacuous to me. It's like reusing the word in its own definition -- and why I described it as circular.

An alternative to this would be to talk about people that are interested in other things that are components of or similar to X -- and therefore a spec that defines X subsumes their interests and standardizes some important element for them to achieve interoperability.

As an example: The intended audience of the 'Fantastic Authorization Token' spec are people who are trying to build interoperable libraries that grant third party applications access to restricted resources.

As a counter example: The intended audience of the 'Fantastic Authorization Token' spec are people who are interested in using Fantastic Authorization Token technology.

Comment (mwherman2000) [reply index.html:261]: @dlongley first, thank you for the dialog and moving this forward

Second, counter example:
> This is a football book (or book about football) that is expected to be of interest to people interested in professional football ...a bit wordy

**Comment by mwherman2000** [comment]
> This PR is also missing appropriate `<a>` tags into definitions as well as has some terminology issues that might cause confusion to some audiences such as "verifiable credential technology" as opposed to something like:
> `systems that utilize the Verifiable Credential<a href="#core-data-model">data model</a>`

FYI: as a writer, these tags aren't something I would add until near the end of the Collaborate phase...

image

File: index.html
Comment (mwherman2000) [reply index.html:261]: > "the intended audience for the VC data model spec is people interested in using VC technology"

@dlongley IMOyour paraphrasing isn't quite right.  Try this:

> "the intended audience for [this document aka Recommendation] is people interested in using VC technology"

I think this makes sense (and the original wording more correct) ...but I will endeavor to refine it some more in the next go-around.

File: index.html
Comment (dlongley) [reply index.html:265]: > Who are these people in real life? ...that is, credential authors and consumers. Who are these people on your VC-based app project team?

Well, I would not say that they are software architects or UX specialists. Neither of these roles seem to fit to me (at least not as top of mind). A data modeler/data engineer would fit better as a "credential author".

> I think we need words (audience roles) that identify with real people and their roles. Further, credential authors and consumers sound like users of a verifiable credential-based system ...not designers and builders of VC-based systems. The later IMO is who we're targeting. Correct?

Yeah, "consumer" may not be a good name here for the reason you mention. A software engineer, application developer, or security specialist may be interested / participate in writing software to consume or verify VCs in the way that I meant.

My point was that I'd think the primary audience of the spec is those authoring VCs (dealing with data modeling and vocabularies, etc.) and the secondary audience would be those creating software to consume and verify them.

File: index.html
Comment (dlongley) [reply index.html:286]: "transformation" would work, given that you should be able to (in some cases) translate from one serialization to another.

File: index.html
Comment (mwherman2000) [reply index.html:265]: > My point was that I'd think the primary audience of the spec is those authoring VCs (dealing with data modeling and vocabularies, etc.) and the secondary audience would be those creating software to consume and verify them.

"authoring VCs" or "designing and building _software systems_ capable of creating, transforming, and verifying VCs?

"authoring VCs" sounds too much like someone who is handcrafting the VC JSON (IMO)

Comment (dlongley) [reply index.html:265]: > "authoring VCs" sounds too much like someone who is handcrafting the VC JSON (IMO)

That's right -- I think they are the primary audience.

The spec's main focus is on data modeling (it's right there in the title) -- and telling this audience how they need to go about modeling their data so it fits the standard, i.e., how to model the issuer, dates, schemas, status, that the fact that a VC is about one or more credential subjects and where to put that information, about how claims are subject+property+value (and how a value can itself be another subject with an identifier and more claims), showing examples on how to do extensions and include contexts, etc. All of this is about how to "craft" a VC so it will be processable by tools written against the standard.

So, yes, I think the primary audience is those people doing the crafting and the secondary audience is those people doing the processing of what's been crafted.

File: index.html
Comment (TallTed) [reply index.html:1287]: It's generally not considered friendly collaboration to respond to a request with a "No" and a "conversation resolved" button click.  Both parties generally must agree that the conversation has been resolved, before that button should be clicked.

File: index.html
Comment (dlongley) [reply index.html:261]: If I were copy editing something that had this statement:

"This is a football book (or book about football) that is expected to be of interest to people interested in professional football ..."

I'd recommend cutting it because it doesn't add value. Of course a football book is expected to be of interest to people interested in football. If the statement is to be of value, it should explain a bit more, for example, this is a football book that is expected to be of interest to people who would like to know more about defensive strategies to employ against well known offensive strategies (or whatever).

I'd say that the goal of mentioning an intended audience for a document is to narrow (or highlight) something that wouldn't otherwise be known from the title of the document or the rest of its basic introduction. If we're not adding more information with the statement, then I'd consider it is a net negative on the document; it makes it more verbose or complex which is an unnecessary burden on readers and/or maintainers. I think we should want to make a specification as short as possible (but not shorter) :).

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-09-15-vcwg#section3-5) on 2021-09-15

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.5. Added intended audience, purpose, goals/non-goals (pr vc-data-model#814)

_See github pull request [#814](https://github.com/w3c/vc-data-model/pull/814)._




> *Brent Zundel:* [https://github.com/w3c/vc-data-model/pull/814](https://github.com/w3c/vc-data-model/pull/814)

**Brent Zundel:** 814 is intended audience purpose and goals and non-goals.  
… This is in response to a couple of issues. This PR adds to the introduction some deliberate language around who the intended audience for the recommendation is or should be, or who we were writing it to.  
… Long conversation back and for there; as well as a purpose statement, for the goals of the specification.  
… I think a number of these additions could be beneficial for the specification. Some of them are not clear to me how necessary they are.  
… I don't see Michael Herman here...  
… I know he and Dave have been having the bulk of the conversation...  
… Dave, would you talk about this PR?  

**Dave Longley:** Sure. My feedback is largely around do we agree on the purpose and intended audience.  
… I thought was added here might be a little broad.  
… I've given that feedback to Michael... I think he said he would address those concerns.  
… I think more people should look at this PR  
… to make sure it matches our intended audience and purpose.  
… I think we should get more approvals on it to accept it.  

**Brent Zundel:** +1, please review.  
… This one in particular, is arguably editorial, but makes a lot of statements.  
… If we don't agree these are statements that we as a group want to make... those are questions we need to address.  

> *Ted Thibodeau Jr.:* +1 more reviews/input/confirmation == more better

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-09-29-vcwg#section1-1) on 2021-09-29

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.1. Added intended audience, purpose, goals/non-goals (pr vc-data-model#814)

_See github pull request [#814](https://github.com/w3c/vc-data-model/pull/814)._




**Wayne Chang:** Michael, can you give us a summary?  

**Michael Herman:** I started a process there, good feedback, responded to some of that feedback. I've incorporated a lot of feedback into local copy, busy w/ other things, can respond to comments that are there.  

**Manu Sporny:** I was waiting for those changes to come in to do a re-review because there was quite a bit of discussion between Michael and Dave.  
… I am concerned about the size of the PR. I agree with some of the content at the beginning but then it gets repetitive and then it gets into a lot of things I have concerns. It may look like we're close here but I haven't done a review and have a lot of concerns, I don't want people to be surprised. My preference would be for this to be dealt with small bite sized PRs.  
… If we are going to deal with this as a fairly large PRs, I think there will be a lot more input in it.  

**Michael Herman:** I accept that and it's all interconnected. I think this is also text that would be in the guide I'm working on. I apologize for the time it's been taking.  
… I think that it's good that it's one piece, this is the kind of text that I expect being in the guide. Apologies for the timing, live on a farm an sometimes can't get back to doing the things I want to do.  

**Wayne Chang:** That it might have another home might also help reduce the size of the text here.  
… Next PR is by DavidC  

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-11-10-vcwg#section3-2) on 2021-11-10

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.2. Added intended audience, purpose, goals/non-goals (pr vc-data-model#814)

_See github pull request [vc-data-model#814](https://github.com/w3c/vc-data-model/pull/814)._




**Manu Sporny:** Michael said he would create a separate document to address his concerns.  
… we should close this PR once this new document is created.  

**David Chadwick:** I agreed to be the 2nd author w/ Michael on his proposed document, but I have not seen a new draft, haven't heard from him in a while..

**Brent Zundel:**  We don't want this to just sit either, been discussed several times in meetings w/ no progress..

**Dave Longley:** suggests we give this a time limit and then close it.  

**Ted Thibodeau Jr.:** we don't want to instigate unhelpful reactions.  

**Brent Zundel:** I will ping Michael and ask him when he might be able to move this forward.  

</details>

**Comment by brentzundel** [comment]
@mwherman2000 if we don't hear from you in the next 7 days with a concrete timeline for making progress, we will be closing this PR.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-11-17-vcwg#section3-1) on 2021-11-17

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.1. Added intended audience, purpose, goals/non-goals (pr vc-data-model#814)

_See github pull request [vc-data-model#814](https://github.com/w3c/vc-data-model/pull/814)._




**Brent Zundel:** I believe we discussed this last time.  
… we don't have any response from Michael Herman who raised the PR.  
… I think the consensus here is that without response from him that we should close this.  
… we have active changes requested and no response with no activity since September.  

**Manu Sporny:** How long do we wait for a non-response to close a PR?.  
… we've waited 7 days and standard wait time has been 14 days.  
… we re-ping him and see if there's any action plan for this.  

**Brent Zundel:** is there any issues with this course of action?.  

**David Chadwick:** can we add a pending close label on it?.  

**Brent Zundel:** yeah we can do that and I'll ping Michael Herman now.  
… pending close label added.  

**Manu Sporny:** we should also remove the other labels if you're ok with that.  

**Brent Zundel:** I'm good with leaving V1.1 editorial but we can remove `possibleErratum` label.  

**Manu Sporny:** Does Editorial label do anything for Errata document?.  

**Kyle Den Hartog:** No only `Errata` label affects that document.  
… I'm good with using only pending close, V1.1, v2.0, 14 day merge, `Errata`, and `possibleErratum` labels.  

</details>

**Pull request closed without merging**
